src/App.tsx

```diff
@@ -83,7 +96,7 @@ const DashboardSidebar = () => {
         )}
         <Sidebar.Item label="Realm settings" link="/settings" icon="cog" />
         <Sidebar.Separator />
-        <Sidebar.ApiStatus healthy={isApiHealthy} realm={astarte.realm} />
+        <Sidebar.ApiStatus healthy={isApiHealthy()} realm={astarte.realm} />
```

Comment:
this invokes the method inside the render. I'd rather assign it to a const variable. This may look like a broken lamba

Reply:
Noted and fixed it, thanks